src/index.test.ts

```diff
@@ -100,20 +100,18 @@ describe("main module", () => {
     assert(
       purgeSpy.calledOnceWithExactly(builtTree, args.delete, barrelName, logger)
     );
-    assert(
-      buildBarrelsSpy.calledOnceWithExactly(
-        destinations,
-        quoteCharacter,
-        semicolonCharacter,
-        barrelName,
-        logger,
-        baseUrl,
-        args.exportDefault,
-        args.structure,
-        args.local,
-        args.include,
-        args.exclude
-      )
+    buildBarrelsSpy.calledOnceWithExactly(
```

Comment:
Not suggesting it needs changing in this PR - but it looks like the assert here has been removed, but retained earlier up (line 89)

Reply:
good catch.. fixed it